src/Concordium/Client/Types/Contract/Info.hs

```diff
@@ -472,3 +545,11 @@ data ContractSigsV2
   { csv2InitSig :: Maybe CS.FunctionSchemaV2 -- ^ Schema for the init function.
   , csv2ReceiveSigs :: Map.Map Text (Maybe CS.FunctionSchemaV2) -- ^ Schemas for the receive functions.
   }
+
+-- |Different from ContractSchemaV3 in that the receiveSigs have a Maybe FunctionSchemaV2.
```

Comment:
This comment is not correct.

Reply:
I think it's correct albeit I do not particularly like the wording of the comments for the other schemas, which this comment follows. But I wanted to be consistent. Here's ContractSchemaV3 for the record:

```
data ContractSchemaV3
  = ContractSchemaV3 -- ^ Describes the schemas of a V1 smart contract.
  { cs3InitSig :: Maybe FunctionSchemaV2 -- ^ Schema for the init function.
  , cs3ReceiveSigs :: Map Text FunctionSchemaV2 -- ^ Schemas for the receive functions.
  , cs3EventSchema :: Maybe SchemaType -- ^ Schemas for the events.
  }
  deriving (Eq, Show, Generic)
```

Should I remove the comment?

Reply:
In terms of the types, the comment is correct. But I guess we could add some explanation as to why this differs.
It is because we want to list all the receive functions, and some of them we have more data for via the schema.

Reply:
Right.

Also, such a comment is pretty bad since it says nothing useful. It does not describe a purpose of the type which is what a comment should generally do.

And I fully realize that many comments are not like that. That is not the reason to perpetuate the badness.

Please add as Kasper suggests.